packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md

```diff
@@ -297,6 +297,107 @@ curl -XPUT 'https://${yourOktaDomain}/api/v1/myaccount/directoryProfile' -H 'Aut
 }
 ```
 
+### Send Phone Challenge
+
+<ApiOperation method="post" url="/idp/myaccount/phones/{id}/challenge"/>
+
+Sends a phone challenge using one of two methods: `SMS` or `CALL`. This request can also handle a resend challenge (retry).
+
+Upon a successful challenge, the user receives a verification code by `SMS` or `CALL`. Send a `POST` request to the `/idp/myaccount/phones/{id}/verify` endpoint to use the verification code to verify the phone number. The verification code expires in 5 minutes.
+
+> **Note:** Do not call the `/idp/myaccount/phones/{id}/challenge` endpoint more often than once every 30 seconds.
+
+#### Required scope and role
+
+An Okta scope of `okta.myAccount.phone.manage` is required to use this endpoint.
+
+> **Note:** Admin users are not allowed to call the `/idp/myaccount/phones/{id}/challenge` endpoint.
+
+#### Required scope and role
+
+#### Request path parameters
+
+| Parameter  | Type   | Description                                       |
+| ---------- | ------ | ------------------------------------------------- |
+| `id` | String | The id of the phone factor. Found in the response when a new phone number is created successfully (`POST /idp/myaccount/phones`) or phone(s) is retrieved (`GET /idp/myaccount/phones`)  |
+
+#### Request query parameters
+
+N/A
+
+#### Request body
+
+This request requires the `method` property as its request body. An optional boolean `retry` property can be used to indicate resend challenge when its value is set to `true`.
+
+| Property | Type                     | Description                          |
+| -------- | -------------------------|--------------------------------------|
+| `method`  | String | The method with which the challenge should be sent, valid values are `SMS` and `CALL` |
+| `retry`  | Boolean | An optional property that indicates whether this is a normal challenge or retry |
+
+#### Response body
+
+Returns an empty response with an HTTP 200 status code.
+
+#### Usage example
+
+##### Request
+
+This request sends a verification code by SMS to the phone number represented by `id`. The request is a normal phone challenge, not a retry.
+
+```bash
+curl -XPOST 'https://${yourOktaDomain}/myaccount/phones/{id}/challenge' -H 'Authorization: SSWS {token}' -H 'Content-Type: application/json' --data '{
```

Comment:
token is not SSWS, is Bearer

Reply:
Good catch, I copied from above and forgot to update it. Thanks.